cpp/src/search/contains_column.cu

```diff
@@ -14,85 +14,22 @@
  * limitations under the License.
  */
 
-#include <hash/unordered_multiset.cuh>
```

Comment:
Is `unordered_mutiset` still be used somewhere else?

Reply:
Nope, my plan was to remove it in a separate PR since there might be tests or CMake changes involved. It turns out it's just a header with no tests, so removing it now.